CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+Version 1.3.0
```

Comment:
might warrant a major version bump

Reply:
I went ahead and merged this in, made a separate follow up pr for this: https://github.com/tilezen/mapbox-vector-tile/pull/118 @benoit9126